packages/google_maps_flutter/google_maps_flutter_ios/lib/src/google_maps_flutter_ios.dart

```diff
@@ -587,6 +622,32 @@ class GoogleMapsFlutterIOS extends GoogleMapsFlutterPlatform {
     GoogleMapsInspectorPlatform.instance =
         GoogleMapsInspectorIOS((int mapId) => _channel(mapId));
   }
+
+  /// Parses cluster data from dynamic json objects and returns [Cluster] object.
+  /// Used by `cluster#onTap` method call handler and inspectors [getClusters] response parser.
+  static Cluster parseCluster(
```

Comment:
Shouldn't this be private?

Reply:
`GoogleMapsInspectorIOS`class is using it and needs it to be public.